src/index.js

```diff
@@ -46,6 +47,8 @@ const exported = {
     MercatorCoordinate,
     Evented,
     config,
+    preloadWorkerPool,
+    removePreloadedWorkerPool,
```

Comment:
We should add documentation for these methods. They'll be helpful for a lot of folks, but they could also be tricky if we don't give some guidance. For example, we'd probably recommend calling `map.remove()` when unmounting a React component, but you probably shouldn't call `mapboxgl.removePreloadedWorkerPool()` at that point in the lifecycle.

Reply:
I think we should reconsider this name. "worker pool" isn't the first them GL JS devs are going to look for when they want to speed up load time in their web app, and also leaks our internal architecture. Instead, I think we could name it `initialize()`, `prewarm()`, or something similar.